docs/YIP/YIP-0003-pretrade-risk-checks.md

```diff
@@ -9,10 +9,27 @@ In Work
 We have to develop the mechanism that will limit users possibility to create orders that will put parties at risk of not been able to cover the net exposure by the colleteral. Exposing the parameters by API so it will be visible and understood by the user of the system. Like a Buying power = colleteral * levarage - open order amount, and Equity = colleteral + net exposure.
```

Comment:
Equity is mentioned only once in this line, there is no PnL mentioned anywhere. So is this YIP has to monitor only buying power or also the PnL of opened positions?